unstructured/nlp/tokenize.py

```diff
@@ -14,37 +19,121 @@
 
 CACHE_MAX_SIZE: Final[int] = 128
 
+NLTK_DATA_URL = "https://utic-public-cf.s3.amazonaws.com/nltk_data.tgz"
+NLTK_DATA_SHA256 = "126faf671cd255a062c436b3d0f2d311dfeefcd92ffa43f7c3ab677309404d61"
+
+
+def _raise_on_nltk_download(*args, **kwargs):
+    raise ValueError("NLTK download disabled. See CVE-2024-39705")
+
+
+nltk.download = _raise_on_nltk_download
+
+
+# NOTE(robinson) - mimic default dir logic from NLTK
+# https://github.com/nltk/nltk/
+# 	blob/8c233dc585b91c7a0c58f96a9d99244a379740d5/nltk/downloader.py#L1046
+def get_nltk_data_dir():
```

Comment:
A docstring would be welcome here. Also type-annotation (`-> str | None`) as that's a critical detail for this function.

A good docstring (in my view) states the _contract_ of the function (not how it fulfills it, like what it "does"), along with any caveats. Basically, as a "black-box", what does this function _promise_?

So in this case perhaps something like:
```python
    """The directory where `nltk` resources are located.

    `None` when the location is not writable.
    """
```

Reply:
Docstring added.